src/audio/CMakeLists.txt

```diff
@@ -70,6 +70,19 @@ if(NOT CONFIG_LIBRARY)
 		target_link_libraries(sof PRIVATE c)
 		target_link_libraries(sof PRIVATE m)
```

Comment:
@lkoenig thanks a lot for this patch! This is really interesting stuff.
Some questions:
* what is the license for your 3rd party libraries
* would is your opinion on creating a generic AEC component and then plugin Google AEC. This will allow to have a common framework for different AEC algorithms.

Reply:
@dbaluta I would be down for point 2 maybe once we better pipeline configuration support. The fact that right now we have to lookup components on the other side of the buffer to identify the playback vs ref is a bit too dangerous to make generic imo.